server/controllers/interactions.js

```diff
@@ -0,0 +1,32 @@
+const axios = require('axios');
+
+const baseURL = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp';
+
+module.exports = {
+  addInteraction: ((req, res) => {
+    const { element, widget, time } = req.body;
+
+    const url = '/interactions';
+    const options = {
+      method: 'post',
+      baseURL,
+      url,
+      data: {
+        element,
+        widget,
+        time,
+      },
+      headers: {
+        Authorization: process.env.API_KEY,
+      },
+    };
+    axios(options)
+      .then((result) => {
+        res.status(201).json(result.data);
+      })
+      .catch((err) => {
+        console.log(err);
+        res.sendStatus(422);
```

Comment:
i'm wondering why the status code for failed post request is 422

Reply:
If I am understanding the Learn page, that error code is given as the appropriate response:

<img width="528" alt="image" src="https://user-images.githubusercontent.com/112882051/215203798-c0981fde-42d0-4bc9-b8cc-17718d359585.png">

Reply:
i see. thanks!